package.json

```diff
@@ -387,6 +387,12 @@
     "embedding-sdk:generate-changelog": "yarn conventional-changelog -p angular --pkg enterprise/frontend/src/embedding-sdk/package.template.json --config enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js --tagPrefix embedding-sdk-",
     "embedding-sdk:publish": "cd ./resources/embedding-sdk && npm publish",
     "embedding-sdk:test-unit": "yarn test-unit enterprise/frontend/src/embedding-sdk/",
+    "embedding-sdk:tsc": "tsc --project tsconfig.sdk.json",
+    "embedding-sdk:clean": "rm -rf resources/embedding-sdk/dist && rm -rf resources/embedding-sdk/.tsbuildinfo",
+    "embedding-sdk:dev": "concurrently -n 'tsc,webpack,fix-types' -c 'blue,green,yellow' 'npm:embedding-sdk:dev:dts' 'npm:embedding-sdk:dev:webpack' 'npm:embedding-sdk:dev:fixup'",
+    "embedding-sdk:dev:dts": "yarn embedding-sdk:tsc --watch --preserveWatchOutput",
```

Comment:
> `preserveWatchOutput `: Whether to keep outdated console output in watch mode instead of clearing the screen every time a change happened.

Ah, this is so that tsc doesn't clear the screen and we can see what went wrong in the types, right?

Reply:
It's for when it's used  with concurrently in the `embedding-sdk:dev` command